HarfangLab/harfanglab/models.py

```diff
@@ -29,3 +29,15 @@ class JobTriggerResult(BaseModel):
     action: str
     creationtime: str
     parameters: Optional[Dict[str, Any]]
+
+
+class HostnameEntry(BaseModel):
+    hostname: str
+    ipaddress: str
+    lastseen: str = Field(..., alias="lastseen")
+    ostype: str = Field(..., alias="ostype")
```

Comment:
My bad. My previous comment was ambiguous.
As you renamed the fields like they are in the API response, you don't need to define an alias (Alias are useful when the name of the field in the model doesn't match the name of the field in the serialized structure)
I suggest removing the second part of the lines:
```suggestion
    lastseen: str
    ostype: str
```